README.md

```diff
@@ -31,7 +31,14 @@ OR, if you are on Windows
 $ COPY env.dist .env
 ```
 
-4) Run the server at the default log level (`'info'`):
+4) Set up your database (currently sqlite3 using knex)
+
+```
+$ knex migrate:latest
```

Comment:
this need not be in this patch but maybe we can have an npm script to runt both commands?